### PR TITLE
feat: adds orientable on transformable image

### PR DESCRIPTION
**Pull request by rbnhtl**
Hi,
Thanks for your package!

Images taken from a mobile device often contains exif data to store the image orientation. Uploading portrait images with orientation informations in the exif data in the advanced image field causes the image to rotate and have a black border.

This PR allows the use of intervention orientable method on TransformableImage, which takes into account the orientation stored in the exif data.

However the exif extension must be enabled on the PHP install. This code throws an exception when calling the orientable method if the extension is not loaded. I also changed the composer.json to include the exif extension in the suggest block. 

Do you want me to update the readme to mention the exif package use ?

This is my first code PR, let me know if there are any problems.

### CODE REVIEW

**Comment by eAvio** [comment]
@rbnhtl Thank you very much. It looks perfect.

**Review by ctessier — CHANGES_REQUESTED** [review]
Hi @rbnhtl,

Thank you very much for the work here and sorry for the delay!

This looks good to me. I will have a closer look at it.
In the mean time, can you please replace `orientable` with the term `rotatable` ? More appropriate to me in this situation.

Thank you again.
I will come back to you.

Clément

File: src/TransformableImage.php [hunk 1]
```diff
@@ -149,6 +176,16 @@ private function cropImage(object $cropperData)
         $this->image->crop($cropperData->width, $cropperData->height, $cropperData->x, $cropperData->y);
     }
 
+    /**
+     * Orientate the image.
+     *
+     * @return void
+     */
+    private function orientateImage()
```
Comment (ctessier) [inline src/TransformableImage.php:184]: Can you please position this method at the very end of the file?
And you can indeed update the documentation to add an example and the optional PHP library.

**Review by ctessier — CHANGES_REQUESTED** [review]

**Comment by ctessier** [comment]
Hi again @rbnhtl,

After giving it some thoughts, I understand that this feature aims at using the exif data of the image to automatically rotate the image to the right orientation.
My suggestion of using the word `rotatable` therefore doesn't sound as appropriate as I thought.

What do you think of using `autoOrientate`? Otherwise, `orientate` can work as it is the term used by Intervention.

Thank you

File: src/TransformableImage.php (hunk 1 above)
Comment (rbnhtl) [reply src/TransformableImage.php]: Hello @ctessier, as you requested I moved the method to the bottom of the file and renamed the public method to autoOrientate.
I also added more details in the method comment, as well as in the requirements section of the readme.

Thank you for the review.

**Comment by PhilippVujic** [comment]
Please add this PR

**Review by ctessier — APPROVED** [review]

**Comment by ctessier** [comment]
Hi @rbnhtl 

Thank you again for your work. This has been added to [v1.3.0](https://github.com/ctessier/nova-advanced-image-field/releases/tag/v1.3.0).

Have a good day
Clément

(Closes #62)

**Comment by shaffe-fr** [comment]
Thanks @rbnhtl!

**Pull request merged**